extensions/elytron-security-ldap/runtime/src/main/java/io/quarkus/elytron/security/ldap/LdapRecorder.java

```diff
@@ -47,7 +54,18 @@ public RuntimeValue<SecurityRealm> createRealm(LdapSecurityRealmRuntimeConfig ru
             ldapSecurityRealmBuilder.addDirectEvidenceVerification(false);
         }
 
-        return new RuntimeValue<>(ldapSecurityRealmBuilder.build());
+        SecurityRealm ldapRealm = ldapSecurityRealmBuilder.build();
+
+        if (runtimeConfig.cache().enabled()) {
+            if (ldapRealm instanceof CacheableSecurityRealm) {
+                ldapRealm = new CachingSecurityRealm(ldapRealm,
+                        new LRURealmIdentityCache(runtimeConfig.cache().size(), runtimeConfig.cache().maxAge().toMillis()));
+            } else {
+                log.warn("LDAP realm not cacheable, ignoring 'cache' configuration");
```

Comment:
```suggestion
                log.warn("LDAP realm is not cacheable, ignoring the 'cache' configuration");
```

Reply:
@michalvavrik Is it reading all right ?

Reply:
sure, thanks

Reply:
@jonasjensen-brolstark Hi, please choose one of the proposed doc updates, either from Michal or the one I just suggested, and then we'll get a build running and the PR merged, thanks

Reply:
is done 🙂